ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/representative-journey.ts

```diff
@@ -0,0 +1,144 @@
+import { EventEmitter, Injectable } from '@angular/core';
+import { JourneyBase } from '../base-journey/journey-base';
+import { RepresentativeSuitabilityModel } from './representative-suitability.model';
+import { RepresentativeStepsOrderFactory } from './representative-steps-order.factory';
+import { RepresentativeJourneySteps } from './representative-journey-steps';
+import { HasAccessToCamera } from '../base-journey/participant-suitability.model';
+
+@Injectable()
+export class RepresentativeJourney implements JourneyBase {
+  static readonly initialStep = RepresentativeJourneySteps.AboutVideoHearings;
+
+  readonly redirect: EventEmitter<RepresentativeJourneySteps> = new EventEmitter();
+
+  stepOrder: Array<RepresentativeJourneySteps>;
+
+  private currentStep: RepresentativeJourneySteps = RepresentativeJourneySteps.NotStarted;
+
+  private currentModel: RepresentativeSuitabilityModel;
+
+  private isDone: boolean;
+
+  constructor(private individualStepsOrderFactory: RepresentativeStepsOrderFactory) {
```

Comment:
`individualStepsOrderFactory`, you can probably just call this `stepsFactory` right? (in either case, individual is probably not right)

Reply:
this is already been addressed in a different branch

Reply:
Would it make more sense to do the rename in here though, as it's a new class, than to make the modification in a different class?